doc-content/enterprise-only/openshift/environment-authoring-con.adoc

```diff
@@ -1,16 +1,43 @@
-[id='environment-authoring-con']
+[id='environment-authoring-con'] 
+ifdef::PAM[]
 = Authoring environment
 You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists of {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes.
+endif::PAM[]
+ifdef::DM[]
+= Authoring or managed server environment
+You can deploy an environment for creating and modifying services using {CENTRAL} and for running them in {KIE_SERVERS} managed by {CENTRAL}. This environment consists of {CENTRAL} and one or more {KIE_SERVERS}.
 
-Depending on your needs, you can deploy either a single authoring environment or a high-availability (HA) authoring environment.
+You can use {CENTRAL} both to develop services and to deploy them to one or several {KIE_SERVERS}. For example, you can deploy test versions of services to one {KIE_SERVER} and production versions to another {KIE_SERVER}.
 
-A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. The {KIE_SERVER} includes an embedded in-memory H2 database engine. This type of environment uses the least possible amount of resources. However, because of the in-memory database, restarting the {KIE_SERVER} pod leads to loss of all process information.
+To avoid accidentally deploying wrong versions to a production {KIE_SERVER}, you can create separate environments to author services (_authoring environment_) and to manage deployment of production services (_managed server environment_). You can use a shared external Maven repository between these environments, so that services developed in the authoring environment are available in the managed server environment. Usually, one {KIE_SERVER} is sufficient for an authoring environment.
```

Comment:
The description is too complicated, it would be good to have there some image where is illustrated this env.
Also to avoid deploying wrong version there is parameter KIE_SERVER_MODE for KIE Server, on manged template this is by default set to "PRODUCTION" so it will not allow to deploy snapshot kjars. [See DM KIE Server template](https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/master/templates/rhdm77-kieserver.yaml#L119) in RHDM auth template this mode is set to "DEVELOPMENT"

Reply:
We have an additional task to do a proper architectural overview, Adriel is currently working on the input. We will try to get a diagram in as a part of that; we don't have the complete information for a diagram right now.

Reply:
(To be clear, that task is probably getting done for 7.7 not 7.6)